frontend/src/components/ProjectServiceAccountRow.vue

```diff
@@ -5,164 +5,134 @@ SPDX-License-Identifier: Apache-2.0
-          <span>Download Kubeconfig</span>
-        </v-tooltip>
-      </v-list-item-action>
-      <v-list-item-action v-if="isServiceAccountFromCurrentNamespace && canGetSecrets" class="ml-1">
-        <v-tooltip top>
-          <template v-slot:activator="{ on }">
-            <v-btn v-on="on" icon @click="onKubeconfig">
-              <v-icon>mdi-eye</v-icon>
-            </v-btn>
-          </template>
-          <span>Show Kubeconfig</span>
-        </v-tooltip>
-      </v-list-item-action>
-      <v-list-item-action v-if="canManageServiceAccountMembers" class="ml-1">
-        <v-tooltip top>
-          <template v-slot:activator="{ on }">
-            <v-btn v-on="on" icon @click.native.stop="onEdit">
-              <v-icon>mdi-pencil</v-icon>
-            </v-btn>
-          </template>
-          <span>Change Service Account Roles</span>
-        </v-tooltip>
-      </v-list-item-action>
-      <v-list-item-action v-if="canManageServiceAccountMembers" class="ml-1">
-        <v-tooltip top>
-          <template v-slot:activator="{ on }">
-            <v-btn v-on="on" icon color="red" @click.native.stop="onDelete">
-              <v-icon v-if="isServiceAccountFromCurrentNamespace">mdi-delete</v-icon>
-              <v-icon v-else>mdi-close</v-icon>
-            </v-btn>
-          </template>
-          <span v-if="isServiceAccountFromCurrentNamespace">Delete Service Account</span>
-          <span v-else>Remove Foreign Service Account from Project</span>
-        </v-tooltip>
-      </v-list-item-action>
-    </v-list-item>
-  </div>
+        </div>
+        <div v-if="isServiceAccountFromCurrentNamespace && canDeleteSecrets" class="ml-1">
+          <v-tooltip top>
+            <template v-slot:activator="{ on }">
+              <v-btn v-on="on" icon @click="onRotateSecret">
+                <v-icon>mdi-refresh</v-icon>
+              </v-btn>
+            </template>
+            <span>Rotate Service Account Secret</span>
+          </v-tooltip>
+        </div>
+        <div v-if="canManageServiceAccountMembers" class="ml-1">
+          <v-tooltip top>
+            <template v-slot:activator="{ on }">
+              <v-btn v-on="on" icon @click.native.stop="onEdit">
+                <v-icon>mdi-pencil</v-icon>
+              </v-btn>
+            </template>
+            <span>Change Service Account</span>
+          </v-tooltip>
+        </div>
+        <div v-if="canManageServiceAccountMembers" class="ml-1">
+          <v-tooltip top>
+            <template v-slot:activator="{ on }">
+              <v-btn v-on="on" icon color="red" @click.native.stop="onDelete">
+                <v-icon v-if="isServiceAccountFromCurrentNamespace">mdi-delete</v-icon>
+                <v-icon v-else>mdi-close</v-icon>
+              </v-btn>
+            </template>
+            <span v-if="isServiceAccountFromCurrentNamespace">Delete Service Account</span>
+            <span v-else>Remove Foreign Service Account from Project</span>
```

Comment:
Why not a computed property?

Reply:
optimized to not use templating conditions